File: SustitucionMOAModel/Models/DataAgro/ContratoAFijar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SustitucionMOAModel.Models.DataAgro
{
    public class ContratoAFijar
    {
        public int CampañaID { get; set; }
        public int TipoNegocioId { get; set; }
        public int Id { get; set; }
        public int MaterialId { get; set; }
        public int Cantidad { get; set; }
        public decimal Precio { get; set; }
        public decimal PrecioNeto { get; set; }
        public int CampanaId { get; set; }
        public string MonedaId { get; set; }
        public int ComercialId { get; set; }
        public string ContratoSAP { get; set; }
        public int PorcentajeDePago { get; set; }
        public int? ComercialCreadorId { get; set; }
        public int EstadoId { get; set; }
        public int ClasificacionId { get; set; }
        public string Observacion { get; set; }
        public int DestinoId { get; set; }
        public int BoletoId { get; set; }
        public int BolsaId { get; set; }
        public int ProveedorId { get; set; }
        public int? CorredorId { get; set; }
        public int LocalidadId { get; set; }
        public int ProvinciaId { get; set; }
        public DateTime FechaOperacion { get; set; }
        public DateTime DesdeFijacion { get; set; }
        public DateTime HastaFijacion { get; set; }
        public DateTime FechaEntrega { get; set; }
        public DateTime FechaDesde { get; set; }
        public DateTime Fecha { get; set; }
        public DateTime FechaHasta { get; set; }
        public int ProveedorCreadorId { get; set; }
        public int StandardDeCalidadId { get; set; }
        public int? CondicionFijacionId { get; set; }
        public int? CantidadCamiones { get; set; }
        public int? ImporteSustentable { get; set; }
        public string MonedaSustentable { get; set; }
        public bool? EstablecimientoPropio { get; set; }
        public bool? Consignatario { get; set; }
        public bool? PlanCanje { get; set; }
        public int? ZonaId { get; set; }

        public string ObservacionTercero { get; set; }
        public bool? CalidadTercero { get; set; }
        public bool? SustentableTercero { get; set; }
        public string ContratoCorredor { get; set; }
        public string ContratoVendedor { get; set; }

    }

}

File: SustitucionMOA/src/app/common/models/contratoAFijar.ts
export class ContratoAFijar {
    public TipoNegocioId: number;
    public Id: number;
    public MaterialId: number;
    public Cantidad: number;
    public Precio: number;
    public PrecioNeto: number;
    public CampanaId: number;
    public MonedaId: string;
    public ComercialId: number;
    public ContratoSAP: string;
    public PorcentajeDePago: number;
    public ComercialCreadorId: number;
    public EstadoId: number;
    public ClasificacionId: number;
    public Observacion: string;
    public CondicionFijacionId: number;
    public DestinoId: number;
    public BoletoId: number;
    public BolsaId: number;
    public ProveedorId: number;
    public CorredorId: number;
    public LocalidadId: number;
    public ProvinciaId: number;
    public StandardDeCalidadId: number;
    public FechaOperacion: Date;
    public DesdeFijacion: Date;
    public HastaFijacion: Date;
    public FechaEntrega: Date;
    public FechaDesde: Date;
    public Fecha: Date;
    public FechaHasta: Date;
    public EstablecimientoPropio: boolean;
    public ZonaId: boolean;
    public Consignatario: boolean;
    public PlanCanje: boolean;
    public ObservacionTercero: string;
    public CalidadTercero: boolean;
    public SustentableTercero: boolean;
    public ContratoCorredor: string;
    public ContratoVendedor: string;
    public CantidadCamiones: number;
    public ImporteSustentable: number;
    public MonedaSustentableId: number;
    public Sustentable: boolean;

    constructor() {
        this.TipoNegocioId = 1;
        this.Id = 0;
    }
}
